packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html

```diff
@@ -1,7 +1,16 @@
 <div data-ovh-alert></div>
 
-<h1 data-translate="dedicated_server_interfaces_title"></h1>
+<div class="row">
+    <h1 data-translate="dedicated_server_interfaces_title" class="col-auto mr-auto"></h1>
+    <oui-guide-menu text="{{::'dedicated_server_interfaces_guides' | translate}}" class="col-auto px-5">
+        <oui-guide-menu-item  href="https://docs.ovh.com/gb/en/dedicated/ola-manager/" external>
```

Comment:
can we put the localized guide in app/config/constants.config.js ?

Reply:
It's only used in one place and not supposed to be translated (not yet).

@lizardK correct me if I'm wrong.

Reply:
Correct. There's only EN version and it won't be translated.